gtsam_unstable/linear/tests/testQPSolver.cpp

```diff
@@ -190,7 +190,7 @@ TEST(QPSolver, iterate) {
     // do not recompute dual variables after every step!!!
     //    CHECK(assert_equal(expected[it], state.values, 1e-10));
     //    CHECK(assert_equal(expectedDuals[it], state.duals, 1e-10));
-    it++;
+    // it++;
```

Comment:
Should we just kill this variable?

Reply:
If I do the commented out tests would fail to compile. Opted to comment out.